### ISSUE TITLE
Docker fails to start after update last night

### ISSUE THREAD

**Issue by BrizzleRocker** [docker/for-win#1074]
### Expected behavior
Docker should start when Windows boots. If not you enter the GUI and restart manually.

### Actual behavior
Docker fails to start immediately and does not respond to a restart either.

### Information

  - Diagnostic ID from "Diagnose & Feedback" in the menu.
2383EC5B-5EC8-432B-97AA-EBDA26D31193/2017-09-07_08-29-17

```
Unable to create: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: The operation failed.

An unexpected error occurred: Logon failure: the user has not been granted the requested logon type at this computer. (0x80070569).

The operation failed.

The Hyper-V Virtual Machine Management service encountered an unexpected error: Logon failure: the user has not been granted the requested logon type at this computer. (0x80070569).
at New-MobyLinuxVM, <No file>: line 201
at <ScriptBlock>, <No file>: line 385
   at Docker.Backend.HyperV.RunScript(String action, Dictionary`2 parameters)
   at Docker.Backend.ContainerEngine.Linux.Start(Settings settings)
   at Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass9_0.<Register>b__0(Object[] parameters)
   at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
```
Windows 10 Pro - version 1703, build 15063.540 (i7-7660 @ 2.5ghz, 16GB ram, 64 bit os)
Docker 17.06.2-ce-win27(13194) - updated last night - channel stable but also happens on edge too.

### Steps to reproduce the behavior
  1. Update a stable working Docker instance.

**Comment by BrizzleRocker** [comment]
Following the instructions [here ](https://github.com/docker/for-win/issues/1066#issuecomment-327569668)I was able to get up and running again. Might be worth updating the documentation to ensure Hyper-V is enabled first?

**Comment by jasonbivins** [comment]
Hi @BrizzleRocker 
Was this a new install, or an update?  
Was Hyper-V broken, or just disabled?

**Comment by metya** [comment]
I have the same problem.
After update I get this error. Ok, I have uninstalled docker and install again with new version installer and get the same error. 

But docker with windows containers if swith to it works fine.

**Comment by BrizzleRocker** [comment]
@jasonbivins Hi there, this was an upgrade to yesterday's stable release. Prior to the upgrade I had experienced issues starting on boot but usually resolved with a restart however that did not fix these issues. As I had a demo at 11am I uninstalled Docker and Hyper-V then restarted and installed Hyper-V followed by Docker. So far it's all been working fine. Thanks for looking.

@metya - try this: uninstall Docker, then uninstall/deselect Hyper-V in Windows Features - when the computer restarts install Hyper-V first, then Docker for Windows.
